python/README.md

```diff
@@ -4,11 +4,40 @@ This library provides a Pythonic API wrapper for the reference Arrow C++
 implementation, along with tools for interoperability with pandas, NumPy, and
 other traditional Python scientific computing packages.
 
-#### Development details
+### Development details
 
 This project is layered in two pieces:
 
 * pyarrow, a C++ library for easier interoperability between Arrow C++, NumPy,
   and pandas
 * Cython extensions and pure Python code under arrow/ which expose Arrow C++
-  and pyarrow to pure Python users
\ No newline at end of file
+  and pyarrow to pure Python users
+
+#### PyArrow Dependencies:
+These are the various projects that PyArrow depends on.
+
+1. **g++ and gcc Version >= 4.8**
+2. **cmake > 2.8.6**
+3. **boost**
+4. **Parquet-cpp**
+
+  The preferred way to install parquet-cpp is to use conda.
+  You need to set the ``PARQUET_HOME`` environment variable to where parquet-cpp is installed.
+  ```bash
+  conda install -y --channel apache/channel/dev parquet-cpp
```

Comment:
This is OK for now. I'd like to get this into conda-forge as soon as possible to build from a shared compiler toolchain vs. the current gcc 4.9 dev builds